app/models/product.rb

```diff
@@ -0,0 +1,2 @@
+class Product < ActiveRecord::Base
```

Comment:
Should we add reference for product, product belongs_to user and product belongs_to m_book ?